externs/shaka/player.js

```diff
@@ -1332,6 +1333,10 @@ shaka.extern.ManifestConfiguration;
  *   The minimum time between recoveries when VIDEO_ERROR is reached, in
  *   seconds.
  *   Defaults to <code>5</code>.
+ * @property {boolean} useActualLiveStreamDuration
+ *   If <code>true</code>, the media source live duration
+ *   set as a<code>Infinity<code>
```

Comment:
typo: `a` before `<code>`
typo: closing tag should be `</code>` not `<code>`

Reply:
The typo issue is fixed.